go.mod

```diff
@@ -36,13 +37,13 @@ require (
 	github.com/form3tech-oss/jwt-go v3.2.3+incompatible // indirect
 	github.com/go-logr/zapr v1.2.0 // indirect
 	github.com/gogo/protobuf v1.3.2 // indirect
+	github.com/golang-jwt/jwt/v4 v4.0.0 // indirect
```

Comment:
We normally don't update the vendor package only if there's a specific reason

Reply:
Removed the update for the vendor package